김은지/주식가격.js

```diff
@@ -0,0 +1,16 @@
+function solution(prices) {
+  const afterPrices = [];
+  const terms = [];
+
+  for (let i = prices.length - 1; i >= 0; i--) {
+    let term = 0;
+    for (let j = afterPrices.length - 1; j >= 0; j--) {
+      term++;
+      if (prices[i] > afterPrices[j]) break;
+    }
+    terms.push(term);
```

Comment:
```suggestion
    terms.push(afterPrices.findLastIndex((afterPrice) => afterPrice >= prices[i]) + 1);
```

프로그래머스는 Node.js 16.17.0 사용하고 있어서 `findLastIndex` 메소드 사용 못했는데, Node.js 18.0.0 부터는 `findLastIndex` 메소드 사용해서 이렇게도 풀 수 있을 것 같아요!